### ISSUE TITLE
Wrong type of level expected for ua100m (100u) and va100m (100v)

### ISSUE THREAD

**Issue by plesager** [EC-Earth/ece2cmor3#742]
Since 97ab257, we ask EC-Earth3 to output ua100m (100u) and va100m (110v) from the 228 table (grib code 246 and 247). The variables appears in the GG files with a **typeOfLevel = surface**. However, `ece2cmor3` expects these variables to be on level type 105  (height_level_code). All of which explain these error message I got:

````
2023-02-14 16:43:07 ERROR:ece2cmor3.grib_filter: Field missing in the first day of file: code 246.228, level type 105, level 100. Dismissing task ua100m in table 6hrPlev
2023-02-14 16:43:07 ERROR:ece2cmor3.grib_filter: Field missing in the first day of file: code 247.228, level type 105, level 100. Dismissing task va100m in table 6hrPlev
````
I haven't dig further, but I expect this to be relatively easy to fix.

**Comment by goord** [comment]
Should be fixed after [6d24d48](https://github.com/EC-Earth/ece2cmor3/commit/6d24d48c74b1176712ac0408c5fc00b7497a78d8)

**Comment by treerink** [comment]
Thanks @goord, yes this is fixed, I have tested it.
